src/configuration/GlobalConfiguration.cpp

```diff
@@ -18,6 +18,14 @@
 #include "MString.h"
 #include <cstdio>
 
+// The exponential moving average is calculated as
+//     ema = current * alpha + previous * (1 - alpha)
+const double GlobalConfiguration::EXPONENTIAL_MOVING_AVERAGE_ALPHA = 0.5;
+
+// Whether to use SoI instead of Reluplex for local search for satisfying assignments
+//to non-linear constraint.
+const bool GlobalConfiguration::USE_DEEPSOI_LOCAL_SEARCH = false;
```

Comment:
why not set this to true by default?

Reply:
Will do after extending each PLConstraint with SoI as discussed off-line